packages/react/src/lib/accessibility/Behaviors/Toolbar/toolbarMenuItemRadioBehavior.ts

```diff
@@ -0,0 +1,37 @@
+import * as keyboardKey from 'keyboard-key'
+
+import { Accessibility } from '../../types'
+import { IS_FOCUSABLE_ATTRIBUTE } from '../../FocusZone'
+
+/**
+ * @specification
+ *  Adds attribute 'aria-checked=true' based on the property 'checked'.
+ *  Adds attribute 'aria-disabled=true' based on the property 'disabled'.
+ *  Adds role='menuitemradio'.
+ *  Adds attribute 'tabIndex=0' to 'root' slot.
+ */
+const toolbarMenuItemRadioBehavior: Accessibility<ToolbarMenuItemRadioBehaviorProps> = props => ({
+  attributes: {
+    root: {
+      [IS_FOCUSABLE_ATTRIBUTE]: true,
+      'aria-checked': props.checked,
+      'aria-disabled': props.disabled,
+      role: 'menuitemradio',
+      tabIndex: 0,
```

Comment:
we should not need tabIndex if data-is-focusable is used and the item is in focus zone

Reply:
Good catch, removed 👍